spec/lib/appsignal_spec.rb

```diff
@@ -92,18 +92,23 @@
 
       context "when allocation tracking and gc instrumentation have been enabled" do
         before do
+          GC::Profiler.stub(:enable)
```

Comment:
Okay, I'll give in, we should follow RSpec's preferred way of doing things.

To gripe on RSpec for a minute: while I agree that using less monkey-patching is a good idea, I feel like dropping the word "stub" for stubbing is a weird idea, and it makes writing expressive test suites in RSpec even more of an uphill battle. </rant>